### PR TITLE
Remove dead links

### PR DESCRIPTION
**Pull request by martinbonnin**
These 2 links were 404 now. Feel free to replace with the new links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,10 +75,6 @@ options.
 
 # Reference Documentation
 
-App Engine Standard Environment:
-* [Using Gradle and the App Engine Plugin (standard environment)](https://cloud.google.com/appengine/docs/java/tools/gradle)
```
Comment (chanseokoh) [inline README.md:79]: These should work:
https://cloud.google.com/appengine/docs/standard/java/using-gradle
https://cloud.google.com/appengine/docs/standard/java/gradle-reference

@GoogleCloudPlatform/cloud-java-team-teamsync the links for the Maven plugin are also broken. But this is weird; I remember these used to work recently. The page has the banner

> Notice: Over the next few months, we're reorganizing the App Engine documentation site to make it easier to find content and better align with the rest of Google Cloud products. The same content will be available, but the navigation will now match the rest of the Cloud products.

and the actual link redirects to `.../docs/legacy/standard/...`. I wonder if the breakage is related.

Comment (chanseokoh) [reply README.md:79]: Ah, they also have "-gen2" docs that I think focus on post-Java 8. So we have these links for "using Gradle."

https://cloud.google.com/appengine/docs/standard/java/using-gradle
https://cloud.google.com/appengine/docs/legacy/standard/java/using-gradle
https://cloud.google.com/appengine/docs/standard/java-gen2/using-gradle

Comment (mpeddada1) [reply README.md:79]: I think you're right, @chanseokoh. Created a PR to fix these links: https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/424

**Comment by elefeint** [comment]
Thanks! This is superceded by #424 and #473.

**Pull request closed without merging**